src/main/java/core/basesyntax/Countable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Countable {
```

Comment:
bad interface name

you don't count anything - you calculate the are of a figure :)

I would rename it to `AreaCalculator`

or you can get rid of this interface at all and add `public abstract double getArea()` to the `Figure` class :)

Reply:
)) I did exactly like you said with 'abstract double getArea()' before but received a recommendation to remove it to a separate interface.